packages/contracts/test/chugsplash/hardhat-tools.spec.ts

```diff
@@ -0,0 +1,326 @@
+import { expect } from '../setup'
+
+/* Imports: External */
+import hre from 'hardhat'
+import { ethers } from 'ethers'
+import { remove0x } from '@eth-optimism/core-utils'
+
+/* Imports: Internal */
+import {
+  ChugSplashActionType,
+  getContractDefinition,
+  makeActionBundleFromConfig,
+} from '../../src'
+import { NON_NULL_BYTES32, NON_ZERO_ADDRESS } from '../helpers'
+
+describe('ChugSplash hardhat tooling', () => {
+  describe('makeActionBundleFromConfig', () => {
+    it('shoulld make a bundle from config with one contract and no variables', async () => {
+      const bundle = await makeActionBundleFromConfig(hre, {
+        contracts: {
+          MyContract: {
+            address: `0x${'11'.repeat(20)}`,
+            source: 'Helper_StorageHelper',
+            variables: {},
+          },
+        },
+      })
+
+      expect(bundle.actions.length).to.equal(1)
+      expect(bundle.actions[0].action).to.deep.equal({
+        actionType: ChugSplashActionType.SET_CODE,
+        target: `0x${'11'.repeat(20)}`,
+        data: getContractDefinition('Helper_StorageHelper').deployedBytecode,
+      })
+    })
+
+    it('should make a bundle from config with two contracts and no variables', async () => {
+      const bundle = await makeActionBundleFromConfig(hre, {
+        contracts: {
+          MyContract1: {
```

Comment:
Are these keys ignored?

Reply:
For L2, currently yes. They're needed for L1.